scripts/buildPublishedAPI.js

```diff
@@ -124,13 +124,18 @@ async function build() {
   let babelPatch = patches.find(name => name.startsWith('@babel'));
   fs.copySync(path.join(__dirname, '..', 'patches', babelPatch), path.join(dir, 'patches', babelPatch));
 
-  // Copy package.json for each package into docs dir so we can find the correct version numbers
+  // Copy from node_modules and pretend it is src
   console.log('moving over from node_modules');
   for (let p of packages) {
     if (!p.includes('spectrum-css') && fs.existsSync(path.join(dir, 'node_modules', p))) {
+      let json = JSON.parse(fs.readFileSync(path.join(dir, 'node_modules', p)), 'utf8');
```

Comment:
I asked Daniel about why this is reading from `node_modules` instead of `packages` like below. He said it was an order of operations issue and existed in `node_modules` at this point of the code. My current understanding is the important change is below where we remove `nightly` builds from ts-differ.